### ISSUE TITLE
Worker died with SIGILL

### ISSUE THREAD

**Issue by hidefog** [pslobo/docker-xmr-node-proxy#2]
Hi,

When I pulled your container from https://hub.docker.com/r/palobo/xmr-node-proxy and ran it, it would constantly give SIGILL errors like this:
```
Received new block template from pool.xxxxx.com
Worker 25 died with code: null, and signal: SIGILL
```

But, after building from the same Dockerfile, it runs just fine.  I'm not sure what the cause was, but I wanted to report it in case that container needs a rebuild or something.

Thanks for putting this together 👍

**Comment by pslobo** [comment]
Hey, thanks for letting me know. Possibly something went wrong with the build. I'll force a rebuild and test.

**Comment by qutimqqcom** [comment]
create  2 times, using
Docker version 17.09.1-ce, build f4ffd2511ce9
always  get 
Worker died with SIGILL


Started server on port: 3333
Received new block template on 325238 height (algo: cryptonight/xtl, variant: xtl) with 2000 target difficulty from eu.XTL.CryptoPool.Space
The proxy currently has 3 miners connected at 0 h/s with an average diff of 2000
Worker 219 died with code: null, and signal: SIGILL
Starting a new worker

and
circle

**Comment by pslobo** [comment]
@qutimqqcom yeah, I possibly need to rebuild the container but haven't had a chance. You can pull the dockerfile and build locally and should be good to go. I'll see what's up with this soon.

**Comment by qutimqqcom** [comment]
> build locally
VM ubuntu  working good

test on board  raspberry pi 3  (os  opensuse leap 42.3 ARM64 +docker version UP post )
 will wait
